saleor/product/tasks.py

```diff
@@ -108,3 +112,9 @@ def _get_preorder_variants_to_clean():
     return ProductVariant.objects.filter(
         is_preorder=True, preorder_end_date__lt=timezone.now()
     )
+
+
+@app.task
```

Comment:
Could you use `queue=settings.UPDATE_INDEX_QUEUE_NAME` so users can use a separate queue for it? By default it should be `None`, I believe that's valid to tell Celery to use the main queue (do check if that's valid)

Reply:
I dont see setting like this anywhere